gapis/api/graph_visualization.go

```diff
@@ -14,6 +14,72 @@
 
 package api
 
+import (
+	"bytes"
+	"fmt"
+)
+
+type Label struct {
+	LevelsName []string
+	LevelsID   []int
+}
+
+func (label *Label) GetSize() int {
```

Comment:
Godocs needed for all public methods.

Reply:
Thank you, I added the Godocs for all public methods.